src/en/charms-storage.md

```diff
@@ -237,6 +238,53 @@ default values:
 juju deploy postgresql --storage pgdata=ebs,10G cache=ebs-ssd
 ```
 
+### Remove storage
+
+If you remove a unit that has storage, the storage is removed with the unit.
+
+The long term plan is to enable storage to be managed separately from units
+for types of storage that can be persistent, such as EBS volumes. This will
+enable the storage to be reused by new units. We have not fully implemented
+this ability. One of the first steps in managing the storage life cycle
+separately is allowing you to remove existing storage, which you can do today.
+
+When you remove the storage attached to a running charm, `remove-storage` will
+run the storage-detaching hook, detach the storage from the machine, and remove
```

Comment:
`storage-detaching`.  Might be worth a link to the hooks page ./reference-charm-hooks#[name]-storage-detaching